core/src/main/java/com/ericsson/bss/cassandra/ecchronos/core/repair/RepairConfiguration.java

```diff
@@ -264,6 +294,18 @@ public Builder withRepairWarningTime(final long repairWarningTime, final TimeUni
             return this;
         }
 
+        /**
+         * Set the initial delay period for a new table job.
+         *
+         * @param initialDelay The time to use as initial delay
+         * @return The builder
+         */
+        public Builder withInitialDelay(final long initialDelay)
```

Comment:
This is not resolved. You should add a TimeUnit parameter. See for instance " withRepairInterval".